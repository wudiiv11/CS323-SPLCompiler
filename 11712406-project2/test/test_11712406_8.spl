int func(int x, int y)
{
  if (x > y) {return 1;}
  else if (y > x) {return -1;}
  else {return 0;}
}

int main()
{
  int a = 5.8;
  int b = 9.2;
  return func(func(a, b), func(b, c));
}