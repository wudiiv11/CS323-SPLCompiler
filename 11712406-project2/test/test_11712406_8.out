Error type 5 at Line 10: unmatching type on both sides of assignment
Error type 5 at Line 11: unmatching type on both sides of assignment
Error type 1 at Line 12: undefined varaible: c
Error type 9 at Line 12: function's args mismatch the declared para